js/gui.js

```diff
@@ -70,24 +74,28 @@ var keyHandlers = {
 
 // On Screen Keys
 $("#leftKey").on("click", function() {
+    swal.close();
     if (player.canMoveLeft()) {
         player.moveLeft();
     }
     player.logPosition();
 });
 $("#rightKey").on("click", function() {
+    swal.close();
     if (player.canMoveRight()) {
         player.moveRight();
     }
     player.logPosition();
 });
 $("#upKey").on("click", function() {
+    swal.close();
     if (player.canMoveUp()) {
         player.moveUp();
     }
     player.logPosition();
 });
 $("#downKey").on("click", function() {
+    swal.close();
```

Comment:
I will accept the style changes but not the code duplication : https://solidsourceit.wordpress.com/2012/08/03/does-source-code-duplication-matter/